src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -587,15 +589,15 @@ for (let i = 0; i < 20; i++) {
 
 </DeepDive>
 
-## Best practices for DOM manipulation with refs {/*best-practices-for-dom-manipulation-with-refs*/}
+## DOM मैनिपुलेशन के लिए सर्वोत्तम प्रथाएँ (Best practices for DOM manipulation with refs)
```

Comment:
```suggestion
## DOM मैनिपुलेशन के लिए सर्वोत्तम प्रथाएँ (Best practices for DOM manipulation with refs) {/*best-practices-for-dom-manipulation-with-refs*/}
```

do not remove heading links which are there in `{}` due to this you were seeing the error while commit

Reply:
reverted